packages/playwright/tests/manual-snapshots.spec.ts

```diff
@@ -1,10 +1,11 @@
 import { test, expect, takeSnapshot } from '../src';
 
-test('multiple snapshots are taken', async ({ page }, testInfo) => {
+test('snapshot names increment for both manual and automatic', async ({ page }, testInfo) => {
   await page.goto('/manual-snapshots');
-  await takeSnapshot(page, 'accordion collapsed', testInfo);
+  await takeSnapshot(page, testInfo);
   await page.locator('summary').click();
   await expect(page.getByText('I am hiding inside!')).toBeVisible();
+  await takeSnapshot(page, testInfo);
```

Comment:
Very minor but let's add a name to this one so we can see that the correct order is kept with a mix of default names and set names.
```suggestion
  await takeSnapshot(page, 'another test', testInfo);
```

Reply:
Sounds good! I'll want this one to be nameless (so we are sure that both automatic and manual snapshots get incremented), but I'll add another manual snapshot with title.

Reply:
That works, although there are two manual snapshots in here already (and the auto-snapshot at the end), so only one of the manual ones would have the name set.

Reply:
Oh, I meant that I wanted to make sure that both manual and automatic showed incrementing (so I needed a manual snapshot to not be titled `Snapshot #1`, in addition to making sure the automatic snapshot was not `Snapshot #1`). Does that make sense?